assets/embauche.html

```diff
@@ -50,7 +50,16 @@
 					<option>mois</option>
 					<option disabled>jour</option>
 					<option disabled>heure</option>
-				</select>.
+				</select>
+
+				sur la commune de
+				<input name="code_postal_entreprise" id="code_postal_entreprise" class="medium-width centered"
+				type="text" placeholder="code postal" pattern="\d{5}" title="Entrez le code postal de l'établissement où le salarié sera employé">
```

Comment:
I've thought about providing a default value... but which one ?
Paris would be the "it can't be worse" value, but would add ~60€ for the median salary...
Ideally, we could compute a mean weighted by the city population of these rates :smile: 
Or we could find a city around 1,5%.

Apart from this choice, I'm afraid seeing this default postal-code could be confusing for the user...